test/ContentHubClientTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Acquia\ContentHubClient\test;
+
+
+use Acquia\ContentHubClient\ContentHubClient;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+
+class ContentHubClientTest extends TestCase
+{
+  /**
+   * @var ContentHubClient
+   */
+  private $contentHubClient;
+
+  /**
+   *
+   */
+  public function setUp() : void
+  {
+    parent::setUp();
+
+    $this->contentHubClient = \Mockery::mock(ContentHubClient::class)
+      ->makePartial();
+  }
+
+  /**
+   *
+   */
+  public function tearDown() : void
+  {
+    parent::tearDown();
+    unset($this->contentHubClient);
+    \Mockery::close();
+    gc_collect_cycles();
```

Comment:
You shouldn't be worried about gc.

Reply:
@looking-4-justice I've read a couple of times that this is a good practice to use gc if you use Mockery. But can delete if you want.

Reply:
Every single line we add to our code/tests adds to the maintenance cost. I, personally, have never seen using this to be a good practice (unless in very rare situations). 
However, if you're comfortable, please put your reference here, so that we all learn something new.

Reply:
Here's the example of this function being used:
https://spiral-framework.com/guide/application-testing
Here's another article, but it's in Ukrainian. Listings are in php though.
https://dou.ua/lenta/articles/php-unit-testing/
Although, once again. I don't want this line to increase the maintenance cost, so I can delete it.

Reply:
Thanks for sharing the links. Honestly, I barely can see any explanation as to why this is required/preferred. Perhaps a matter of the writers' taste. So, please remove it, especially because PHP takes care of these things automatically and gc is on by default.

Reply:
done